### ISSUE TITLE
"alter table keyspace1.standard1 with WITH compression = {'class': 'LZ4Compressor'};" is not working anymore

### ISSUE THREAD

**Issue by raphaelsc** [scylladb/scylla#7321]
head: fd1dd0e

```
$ ./bin/cqlsh -e "alter table keyspace3.standard1 WITH compression = {'class': 'LZ4Compressor'};"
:1:ConfigurationException: Missing sub-option 'sstable_compression' for the 'compression' option.
```

This used to work, but it's no longer working. Intentional or regression?

**Comment by slivne** [comment]
work on cassandra 3.11.3 ...

**Comment by slivne** [comment]
It does not seem to work on 3.0 - so its old ... 

I am not sure we ever supported this 

https://github.com/scylladb/scylla/issues/2061

**Comment by nyh** [comment]
So @slivne, shouldn't this new issue be closed as a dup of #2061?
Sure would have been nice to have a test which reproduces this (and various "with compression" settings which can be tested to work on Cassandra vs. Scylla).

**Comment by slivne** [comment]
yes
